python/bindings.cpp

```diff
@@ -310,8 +316,15 @@ void bindReportReader(py::module& m, const std::string& prefix) {
                                    (prefix + "DataFrame").c_str(),
                                    "Something easily convertible to pandas dataframe")
         .def_readonly("ids", &DataFrame<KeyType>::ids)
-        .def_readonly("data", &DataFrame<KeyType>::data)
-        .def_readonly("times", &DataFrame<KeyType>::times);
+        .def_property_readonly("data",
+                               [](DataFrame<KeyType>& dframe) {
+                                   return asArray(std::move(dframe.data),
+                                                  dframe.n_cols,
+                                                  dframe.n_rows);
+                               })
```

Comment:
just consistency with previous style
```suggestion
        .def_property_readonly("data", [](DataFrame<KeyType>& dframe) {
            return asArray(std::move(dframe.data), dframe.n_cols, dframe.n_rows);
        })
```